src/core/qgsmaplayer.h

```diff
@@ -675,6 +681,20 @@ class CORE_EXPORT QgsMapLayer : public QObject
      */
     QgsMapLayerStyleManager *styleManager() const;
 
+    /**
+     * Sets 3D renderer for the layer. Takes ownership of the renderer.
+     * \note not available in Python bindings
```

Comment:
won't this need a SIP_SKIP then? (same with setRenderer3D)

But also - why not?

Reply:
Yeah the note should not be there... fixing